.github/workflows/build_servers_prerenewal.yml

```diff
@@ -0,0 +1,70 @@
+name: Build servers in Pre-Renewal
```

Comment:
Guess this one isn't needed so much since #6520 will handle the running of the servers? Plus `build_servers_gcc` and `build_servers_msbuild` will handle the compiling process. Unless this will be overtaken with SQL.